crates/factor-sqlite/tests/factor_test.rs

```diff
@@ -62,26 +62,21 @@ async fn no_error_when_database_is_configured() -> anyhow::Result<()> {
     let factors = TestFactors {
         sqlite: SqliteFactor::new(test_resolver),
     };
-    let env = TestEnvironment::default_manifest_extend(toml! {
-        [component.test-component]
-        source = "does-not-exist.wasm"
-        sqlite_databases = ["foo"]
-    });
     let runtime_config = toml! {
         [sqlite_database.foo]
         type = "spin"
     };
     let sqlite_config = SpinSqliteRuntimeConfig::new("/".into(), "/".into());
-    if let Err(e) = env
-        .build_instance_state(
-            factors,
-            TomlRuntimeSource::new(&runtime_config, sqlite_config),
-        )
+    let env = TestEnvironment::new(factors)
```

Comment:
```suggestion
    let _ = TestEnvironment::new(factors)
```
I'm surprised this doesn't lead to a warning?

Reply:
`env` is used at 77